app/src/main/res/layout/search_fragment.xml

```diff
@@ -14,6 +14,14 @@
             app:title="@string/search_label"
             android:id="@+id/toolbar"/>
 
+    <com.google.android.material.chip.Chip
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/toolbar"
+        android:id="@+id/feed_title_chip"
+        android:layout_marginLeft="10dp"
+        android:layout_marginRight="0dp"/>
```

Comment:
Please use the same indention as the rest of the file (eg. make this 12 spaces instead of 8)